bindings/python/tests/test_api_responses.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Generic, TypeVar
+from json import load, loads, dumps
+from iota_sdk import RoutesResponse, CongestionResponse, ManaRewardsResponse, ValidatorResponse, ValidatorsResponse, CommitteeResponse, IssuanceBlockHeaderResponse, Block, OutputResponse, SlotCommitment
+
+
+base_path = '../../sdk/tests/types/api/fixtures/'
+T = TypeVar("T")
+
+
+def test_api_responses():
```

Comment:
Just curious, we don't have `SubmitBlockResponse` in Python?

Reply:
I guess bindings core just return the ID

Reply:
Correct